dockerfile/cuda11.1.1.dockerfile

```diff
@@ -36,6 +36,10 @@ RUN apt-get update && \
     util-linux \
     vim \
     wget \
+    apt-transport-https \
+    ca-certificates \
+    gnupg \
+    lsb-release \
```

Comment:
sort in order?

Reply:
Sure...

Reply:
Removed. These packages are followed https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository